lua/tests/dotenv/parser.lua

```diff
@@ -44,21 +44,24 @@ TEST_EMPTY= #This should be treated as nil
 =-100
  =Broken
     =.19
-
+TEST_HASHTAG_IN_QOUTES="This should not #break"
+TEST_WORST_CASE_VALUE="Hello \", how are you # today? I'm good thanks!"
 #End of .env...]]
 
-local EXPECTED_KEYS = {
-	TEST_SINGLE_QUOTES = true,
-	TEST_DOUBLE_QUOTES = true,
-	TEST_STRING = true,
-	TEST_STRING_2 = true,
-	TEST_NUMBER = true,
-	TEST_INTEGER = true,
-	TEST_UPPER_BOOLEAN = true,
-	TEST_LOWER_BOOLEAN = true,
-	TEST_MIXED_BOOLEAN = true,
-	TEST_VALUE = true,
-	TEST_VALUE_QUOTED = true,
+local EXPECTED_PAIRS = {
```

Comment:
Revert these test changes and add new steps for the two keys you added (Ln47-48). While a catch-all case does work, it doesn't clearly outline the specific requirements we're testing against.